node/src/braid/mod.rs

```diff
@@ -248,7 +254,70 @@ impl Braid {
             }
         }
     }
+    /// utility function for GetBeadsAfter request
+    pub fn get_beads_after(&self, old_tips: Vec<BeadHash>) -> Option<Vec<Bead>> {
+        let old_tips: HashSet<BeadHash> = old_tips.into_iter().collect();
+        tracing::warn!(
+            old_tips=?old_tips,"Tips received from the peer for which beads are requested for during IBD"
```

Comment:
I don't understand what this means. How do you know we're in IBD? Are you saying a peer is requesting tips but we're in IBD?